compiler/mir/src/Graph.cpp

```diff
@@ -44,9 +44,14 @@ std::vector<Operation *> getSortedNodes(Graph *graph)
   std::deque<Operation *> ready_nodes;
   std::unordered_map<Operation *, std::size_t> num_visited_input_edges;
 
+  for (Operation *op : graph->getInputs())
+  {
+    ready_nodes.push_back(op);
+  }
```

Comment:
I added this because below `getNodes()` return `unordered_set`, and below searching changes input order and test fail on ubuntu 22.04.  (ex. `compiler/mir2loco/src/mir2loco.test.cpp`'s `Add_Test`)

It may related with different std lib implementation.

@seanshpark Is it acceptable update?

Reply:
> Is it acceptable update?

`mir` project has not been updated more than a year... I think it is OK
@binarman PTAL